src/components/SearchAndFilter/SearchAndFilter.scss

```diff
@@ -111,4 +111,32 @@
     top: 3px;
     z-index: 999;
   }
+
+  .p-search-box {
```

Comment:
That's probably something to update later (maybe when we upstream this to Vanilla CSS), but it seems that if so much of this needs to be overridden, this shouldn't really be using `p-search-box` class at all and be something like `p-search-and-filer__search`.

But now that it's scoped this can wait I guess not to introduce to many changes to the component.